Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml

```diff
@@ -260,17 +262,18 @@
     highPressureMultiplier: 0.5
     lowPressureMultiplier: 1000
   - type: ExplosionResistance
-    damageCoefficient: 0.4
+    damageCoefficient: 0.35
   - type: Armor
     modifiers:
-      coefficients:
-        Blunt: 0.5
-        Slash: 0.6
-        Piercing: 0.6
+        Blunt: 0.35
+        Slash: 0.35
+        Piercing: 0.35
+        Radiation: 0.7
+        Heat: 0.35
         Caustic: 0.7
```

Comment:
Missing `coefficients:` causes fails.
Seems you removed an extra line by mistake